### DISCUSSION TITLE
transfer local issues to organization discussions

### DISCUSSION THREAD

**Discussion by knmcguire** [community/community#17208] [Discussions]
Hi!

Currently if we want to transfer issues of a local repository to the organization discussions, we will need to first transfer the issue to the issue list of the 'discussion' repository, and then make the issue into an discussion. So it always requires 2 steps.

It would be great if the local issues can be immediately transferred to the organizational discussions, so a diagonal transfer. Hope this will be possible in the future!

**Comment by ghost** [comment]
Hi knmcguire. I wanted this feature as the same problem happened to me. I have a local repository and would like to submit some issues to the organization I am subscribed to.
